go/config/enclave_config.go

```diff
@@ -14,6 +14,8 @@ type EnclaveConfig struct {
 	HostAddress string
 	// The address on which to serve requests
 	Address string
+	// Whether the enclave is an aggregator (as opposed to a validator)
+	IsAggregator bool
```

Comment:
I'd say it would be clearer if we made this an "enum"

`NodeType NodeType`

Where NodeType is one of (`Validator`, `Aggregator`)

Not that we'll add more, but it will make the if conditions more clear